content/en/docs/setup/additional-setup/dual-stack/index.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Install Istio in Dual-Stack mode
+description: Install and use Istio in Dual-Stack mode running on a Dual-Stack Kubernetes cluster.
+weight: 70
+keywords: [dual-stack]
+owner: istio/wg-networking-maintainers
+test: no
```

Comment:
Any reason to not enable testing here?

Reply:
As mentioned in the PR description, to enable test for this, I need a way to setup a dual-stack k8s kind cluster. So I am trying to figure that out currently, once done, I will add tests in a separate PR

Reply:
Oops. I missed that comment.